test/sqllogictest/transform/lifting.slt

```diff
@@ -496,7 +501,10 @@ explain select distinct a1.a+2, a1.literal from (select distinct a+1 as a, 123 a
 ----
 %0 =
 | Get materialize.public.t (u1)
-| Distinct group=(((#0 + 1) + 2))
+| Map (#0 + 1)
+| Distinct group=(#2)
+| Map (#0 + 2)
+| Distinct group=(#1)
```

Comment:
I think you mentioned this as a potential bit of fall out. But I forget the details, but would you mind filing an issue as this lands?